webapp/src/monacoEditAmendments.ts

```diff
@@ -101,8 +101,11 @@ function applyAmendment(amendment: EditAmendmentInstance, editor: monaco.editor.
         // determine where to apply
         let amendRange = Object.assign({}, amendment.range)
```

Comment:
u can probably get rid of this object.assign if you're going to do a spread below